extensions/grpc/runtime/src/main/java/io/quarkus/grpc/runtime/config/GrpcConfiguration.java

```diff
@@ -4,28 +4,30 @@
 
 import io.quarkus.runtime.annotations.ConfigDocMapKey;
 import io.quarkus.runtime.annotations.ConfigDocSection;
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefaults;
 
 /**
  * gRPC configuration root.
  */
+@ConfigMapping(prefix = "quarkus.grpc")
 @ConfigRoot(phase = ConfigPhase.RUN_TIME)
-public class GrpcConfiguration {
+public interface GrpcConfiguration {
 
     /**
      * Configures the gRPC clients.
      */
-    @ConfigItem
-    @ConfigDocSection(generated = true)
+    @WithDefaults
```

Comment:
Where was the default client before?

Reply:
Oh, I don't think we had a default client based on the configuration. I'll remove it.